src/util/util.ts

```diff
@@ -37,3 +37,9 @@ export const isSubstringOf = (substring: string, source: string): boolean =>
   pipe(source, S.toLowerCase, S.includes(S.toLowerCase(substring)))
 
 export const isError = (value: unknown): value is Error => value instanceof Error
+
+export const escapeRegExp = (str?: string): string => {
```

Comment:
I wouldn't have called the function like this because it suggests that the escaping is to put safely a string into a regexp, which is not the case (even if the escaped symbols overlap). Instead, I would have called thefunction:  `escapeSparqlStr`.